tests/api_connexion/schemas/test_event_log_schema.py

```diff
@@ -0,0 +1,126 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from airflow import DAG
+from airflow.api_connexion.schemas.event_log_schema import (
+    EventLogCollection, event_log_collection_schema, event_log_schema,
+)
+from airflow.models import Log, TaskInstance
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.utils import timezone
+from airflow.utils.session import create_session, provide_session
+
+
+class TestEventLogSchemaBase(unittest.TestCase):
+
+    def setUp(self) -> None:
+        with create_session() as session:
+            session.query(Log).delete()
+        self.default_time = "2020-06-09T13:00:00+00:00"
+        self.default_time2 = '2020-06-11T07:00:00+00:00'
+
+    def tearDown(self) -> None:
+        with create_session() as session:
+            session.query(Log).delete()
+
+    def _create_task_instance(self):
+        with DAG('TEST_DAG_ID', start_date=timezone.parse(self.default_time),
+                  end_date=timezone.parse(self.default_time)) as dag:
+            DummyOperator(task_id="TEST_TASK_ID", owner="airflow")
+        return TaskInstance(task=op1, execution_date=timezone.parse(self.default_time))
```

Comment:
Whoops my bad, static checks failed

```suggestion
            op1 = DummyOperator(task_id="TEST_TASK_ID", owner="airflow")
        return TaskInstance(task=op1, execution_date=timezone.parse(self.default_time))
```


```suggestion
        with DAG('TEST_DAG_ID', start_date=timezone.parse(self.default_time),
                  end_date=timezone.parse(self.default_time)):
            DummyOperator(task_id="TEST_TASK_ID", owner="airflow")
        return TaskInstance(task=op1, execution_date=timezone.parse(self.default_time))
```